src/host/proxy/Host.Proxy.vcxproj

```diff
@@ -57,8 +57,9 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <AdditionalIncludeDirectories>..;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <CallingConvention>StdCall</CallingConvention> <!-- Must be Stdcall on all platforms to resolve _ObjectStublessClient3 -->
-      <PreprocessorDefinitions>REGISTER_PROXY_DLL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <CallingConvention>StdCall</CallingConvention>
+      <!-- Must be Stdcall on all platforms to resolve _ObjectStublessClient3 -->
+      <PreprocessorDefinitions>REGISTER_PROXY_DLL;WIN32;%(PreprocessorDefinitions)</PreprocessorDefinitions>

```

Comment:
Do you want this defined for everything? Or just some flavors (e.g. ARM64)?

Reply:
Since minwindefs defines it for everyone if it didn't already exist, I suspect there's no harm in us doing it for every flavor of everything :smile: Good catch though!